components/metal/TheDarkHeader.vue

```diff
@@ -0,0 +1,282 @@
+          <radialGradient
+            id="paint2_radial"
+            cx="0"
+            cy="0"
+            r="1"
+            gradientUnits="userSpaceOnUse"
+            gradientTransform="translate(47.954 21.4228) scale(40.0739 30.6688)"
+          >
+            <stop offset="0.2" stop-color="white" stop-opacity="0" />
+            <stop offset="1" stop-color="white" />
+          </radialGradient>
+          <radialGradient
+            id="paint3_radial"
+            cx="0"
+            cy="0"
+            r="1"
+            gradientUnits="userSpaceOnUse"
+            gradientTransform="translate(23.1794 1.22006) rotate(65.799) scale(26.9268 33.7896)"
+          >
+            <stop offset="0.3406" stop-color="white" stop-opacity="0" />
+            <stop offset="0.8886" stop-color="white" />
+          </radialGradient>
+        </defs>
+      </svg>
+      <h1 class="the-dark-header__title">
+        Qiskit
+      </h1>
+
+      <div class="the-dark-header__media">
+        <img class="the-dark-header__media-cryo" src="/images/metal/hero/cryo.png" alt="">
+        <img class="the-dark-header__media-transmon-outline" src="/images/metal/hero/transmon.svg" alt="">
+        <img class="the-dark-header__media-transmon" src="/images/metal/hero/transmon.png" alt="">
+      </div>
+    </div>
+  </header>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+import { Component } from 'vue-property-decorator'
+import LandingCta from '~/components/landing/LandingCta.vue'
+import AppLink from '~/components/ui/AppLink.vue'
+
+@Component({
+  components: { LandingCta, AppLink }
+})
+export default class extends Vue {}
+</script>
+
+<style lang="scss">
+@import '~carbon-components/scss/globals/scss/typography';
+
+.the-dark-header {
+  @include responsive-grid-bg-strip('/images/grid/grid-hero-dark.svg', auto, 56rem);
+
+  position: relative;
+  overflow: hidden;
+  // In Figma, the height is not enforced but the background is always
+  // visible completely so we do it in the CSS. A small correction is needed
+  // to be able of displaying the bottom lines of the grid.
+  height: calc(56rem + 2px);
+
+  @include mq($from: medium, $until: large) {
+    // To adjust to the size of the smaller grid.
+    height: calc(#{56rem * 40 / 64} + 2px);
+  }
+
+  @include mq($until: medium) {
+    height: calc(#{56rem * 40 / 64} + 2px);
+  }
+
+  &__nav {
+    @include contained();
+
+    &-link {
+      color: $white;
+      display: flex;
+      font-size: 1rem;
+      position: absolute;
+      top: $layout-01;
+      z-index: 2;
+    }
+
+    &-icon {
+      fill: currentColor;
+      margin-right: $spacing-05;
+    }
+  }
+
+  &__container {
+    @include contained();
+
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding-top: 23rem;
+
+    @include mq($from: medium, $until: large) {
+      padding-top: 12.75rem;
+    }
+
+    @include mq($until: medium) {
+      padding-top: 14.15rem;
+    }
```

Comment:
Same here with another width:
```suggestion
    @include mq($until: medium) {
      padding-top: 14.15rem;
      max-width: 450px;
    }
```

Reply:
Same here, I think we can move forward without the `max-width`, so that the page title can align left with the rest of the content.